lang/ghc/Portfile

```
@@ -350,10 +364,15 @@ subport ghc-prebuilt {
         }
     }
 
+    depends_lib-append \
+                    port:cctools \
+                    port:ld64 \
+                    port:llvm-${ghc_llvm_version}
+
     extract.only    ${ghc_distname}${extract.suffix}
-if {[exists extract.rename]} {
-    extract.rename  yes
-}
+    if {[exists extract.rename]} {
+        extract.rename  yes
+    }
```

The reason it was indented that way was that `if {[exists extract.rename]} {` and `}` were intended to be temporary, only needed during the brief transition period leading up to and shortly after the January 2023 release of MacPorts 2.8.1 which introduced the `extract.rename` option. Now that that release has been out for well over two weeks (a year, in fact), the `if` statement should be removed, leaving only the unconditional `extract.rename yes`.
```suggestion
    extract.rename  yes
```

---

Thanks. I implemented this, along with some accompanying improvements to PG `haskell_cabal`.